resources-ktx/src/main/kotlin/Text.kt

```diff
@@ -40,6 +40,10 @@ public sealed class Text : Parcelable {
     public data class Resource(@StringRes public val resourceId: Int) : Text() {
         override fun get(context: Context): String = context.getString(resourceId)
     }
+
+    public companion object {
+        public val EMPTY: Plain = Plain("")
```

Comment:
Maybe better to return `Text` here instead of `Plain`?

Reply:
And it would be great to add KDoc comment to this field

Reply:
Did I use the right formatting style?

Reply:
Ok, that's good